shell/docker-push.sh

```diff
@@ -2,5 +2,9 @@
 # Ensure we fail the job if any steps fail
 set -e -o pipefail
 
+
```

Comment:
The entire change to this file should be removed. What is the need for four extra blank lines here?

Reply:
This change was made when the JJB Verify wasn't getting triggered. IT asked to create a dummy change to some file for the very job  to get triggered.